test/unit/cmap/connection_pool.test.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+require('util.promisify/shim')();
```

Comment:
Minor nitpick, but I'd put this next to the `require('util')` just for the sake of organization.

Reply:
Hey, this was brought closer to the util call.

Reply:
Thanks!